mkosi/resources/mkosi.md

```diff
@@ -666,11 +667,12 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 : Configure compression for the resulting image or archive. The argument can be
   either a boolean or a compression algorithm (`xz`, `zstd`). `zstd`
   compression is used by default, except CentOS and derivatives up to version
-  8, which default to `xz`. Note that when applied to block device image types,
+  8, which default to `xz`, and OCI images, which default to `gzip`.
```

Comment:
Before anyone asks, OC**I** **i**mages is correct, as the first abbreviation stands for open container initiative